server/controllers/connectedAccounts.js

```diff
@@ -131,6 +131,26 @@ export const createOrUpdate = (req, res, next, accessToken, data, emails) => {
   }
 };
 
+export const disconnect = async (req, res) => {
+  const { collectiveId: CollectiveId, service } = req.params;
+  const { remoteUser } = req;
+
+  try {
+    const Account = await ConnectedAccount.findOne({
+      where: { service, CollectiveId },
+    });
+
+    await Account.delete(remoteUser);
```

Comment:
Let's default to camel case for variables

```suggestion
    await account.delete(remoteUser);
```

Reply:
resolved with - https://github.com/opencollective/opencollective-api/pull/2672/commits/ee8a906d7dc974721f519c49107488d3d31990d5